app/services/jobs/return-logs/fetch-licence-return-requirements.service.js

```diff
@@ -0,0 +1,128 @@
+'use strict'
+
+/**
+ * Fetches data needed for the generating return logs
+ * @module FetchReturnLogsService
+ */
+
+const ReturnLogModel = require('../../../models/return-log.model.js')
+const ReturnRequirementModel = require('../../../models/return-requirement.model.js')
+const ReturnVersionModel = require('../../../models/return-version.model.js')
+
+const { db } = require('../../../../db/db.js')
+const { cycleEndDate, cycleStartDate, startOfWinterAndAllYearCycle } = require('../../../lib/dates.lib.js')
+
+/**
+ * Given the licence reference this service returns the return requirements to be turned into return logs.
+ *
+ * @param {string} licenceReference - if provided only do the return log for that licence
+ *
+ * @returns {Promise<Array>} the array of return log payloads to be created in the database
+ */
+async function go (licenceReference) {
+  const returnRequirements = await _fetchReturnRequirements(licenceReference)
+
+  return returnRequirements
```

Comment:
We can return directly here

```suggestion
  return _fetchReturnRequirements(licenceReference)
```